build/ci/build-image/cloudbuild.yaml

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 
 steps:
+  - name: ubuntu
```

Comment:
Wrong cloudbuild.yaml, you want the one in the root of the repository. This one gets run on a chron once a day.